acquisitions/src/main/resources/thunderjet/cross-modules/cross-modules-junit.feature

```diff
@@ -18,13 +18,14 @@ Feature: cross-module integration tests
       | 'orders-storage.module.all'                                 |
 
     * table userPermissions
-      | name                                                        |
-      | 'invoice.all'                                               |
-      | 'orders.all'                                                |
-      | 'finance.all'                                               |
-      | 'orders.item.approve' |
-      | 'orders.item.reopen'  |
-      | 'orders.item.unopen'  |
+      | name                       |
+      | 'invoice.all'              |
+      | 'orders.all'               |
+      | 'finance.all'              |
+      | 'orders.item.approve'      |
+      | 'orders.item.reopen'       |
+      | 'orders.item.unopen'       |
+      | 'inventory.items.item.get' |
```

Comment:
adding this permission here means that it was missed in module descriptor in mod-orders